spec/features/topical_event_spec.rb

```diff
@@ -110,6 +110,7 @@
 
       within("#detailed-guides") do
         related_guidance.each { |title, link| expect(page).to have_link(title, href: link) }
+        expect(page).to have_link("See all detailed guides", href: "/search/guidance-and-regulation?topical_events%5B%5D=something-very-topical")
```

Comment:
Could this change be fixed up to the previous commit, as that's when the test was first introduced